graph_builder.py

```diff
@@ -1,51 +0,0 @@
-# -*- coding: utf-8 -*-'
-import networkx as nx
```

Comment:
Отпала
--
Envoyé de l'application Mail.Ru pour Android vendredi, 09 septembre 2016, 03:49PM +03:00 de Cadovvl < notifications@github.com> :

>In  graph_builder.py :
>> @@ -1,51 +0,0 @@
> -# -*- coding: utf-8 -*-'
> -import networkx as nx
>Это переехало, или отпала необходимость?
>—
>You are receiving this because you authored the thread.
>Reply to this email directly,  view it on GitHub , or  mute the thread .